src/components/pages/Profile/History/PublishedList.tsx

```diff
@@ -59,7 +59,7 @@ export default function PublishedList({
       await getPublished()
     }
     fetchPublishedAssets()
-  }, [accountId, page, appConfig.metadataCacheUri, chainIds, newCancelToken])
```

Comment:
Not sure if this change is responsible but for me the published list on profile does not work properly anymore. In this case in my screenshot there should be no assets shown as there are none on BSC. But whatever I select I always get the ETH mainnet asset shown

![image](https://user-images.githubusercontent.com/90316/135252415-294615d9-bfac-418d-b27d-6075400d259d.png)

Reply:
The same thing seems to be happing on `main` so it doesn't seem to be anything to do with this PR. 

![Network not updating](https://user-images.githubusercontent.com/20739535/135259290-5ec3d229-3b47-4edf-afec-ba44c812eaf1.png)


It seems to be an issue with the state not being updated. I found that if I refresh the page after selecting a different network, it starts showing the correct assets.

![Network not updating 2](https://user-images.githubusercontent.com/20739535/135260036-db90aaff-48cc-47b6-ad7e-21edac76d60e.png)

Reply:
I've created a [seperate issue](https://github.com/oceanprotocol/market/issues/887) for that bug

Reply:
Nope, this is something else and unrelated to this PR. On `main` everything works as expected on the profile page so this specifically is caused by a change in here 
![D4A137AC-1259-432E-B0D1-2979B02FF63C](https://user-images.githubusercontent.com/90316/135265389-2bb9affe-a486-4be6-9c81-d8f98850801c.png)

Reply:
I just added back the chainIds to the dependency list. That should be the only change that affects the results on Published tabs.

Reply:
Looks like that was it. It's fixed now.